src/button/button-group.scss

```diff
@@ -106,3 +106,8 @@
     }
   }
 }
+
+@mixin iui-button-group-overflow-x {
```

Comment:
so when would you apply that class in react?

Reply:
`{ 'iui-button-group-overflow-x': !!overflowButton && orientation === 'horizontal' }`

Reply:
and if you always keep it applied, whats breaking?

Reply:
But if I remember correctly, wehad 100% before your previous changes...no?